.github/workflows/release.yml

```diff
@@ -281,14 +281,14 @@ jobs:
       - name: Build CDDA (linux)
         if: runner.os == 'Linux' && matrix.mxe == 'none' && matrix.android == 'none'
         run: |
-          make -j$((`nproc`+0)) TILES=${{ matrix.tiles }} SOUND=${{ matrix.tiles }} RELEASE=1 LOCALIZE=1 LANGUAGES=all BACKTRACE=0 PCH=0 bindist
+          make -j$((`nproc`+0)) TILES=${{ matrix.tiles }} SOUND=${{ matrix.sound }} RELEASE=1 LOCALIZE=1 LANGUAGES=all BACKTRACE=0 PCH=0 bindist
```

Comment:
I think the `sound` flag is supposed to enable shipping soundpacks with the game, not enabling the sound feature, see #57138.

Reply:
I don't get it. Why we need to include the sound feature when there is no sound packs.

Reply:
Because the player can download a different soundpack themselves afterwards

Reply:
ok, seems this PR is not useful.😅

Reply:
The changes to Makefile are still useful though in case someone wants to compile with sound off themselves.

Reply:
I see. That will require two separate options for sound feature and sound pack, and should be in another PR.